packages/logger/.eslintrc

```diff
@@ -0,0 +1,16 @@
+{
+    "extends": ["../../.eslintrc"],
+    "parserOptions": { "project": "./tsconfig.json" },
+    "rules": {
+        "jsdoc/require-returns": 0,
+        "jsdoc/require-param": 0,
+        "jsdoc/require-jsdoc": 0,
+        "jsdoc/match-description": 0,
+        "jsdoc/multiline-blocks": 0,
+        "jsdoc/newline-after-description": 0,
+        "jsdoc/require-param-description": 0,
+        "jsdoc/no-multi-asterisks": 0,
+        "jsdoc/check-tag-names": 0,
+        "@typescript-eslint/no-unused-vars": ["error", { "varsIgnorePattern": "^_", "argsIgnorePattern": "^_" }]
+    }
+}
```

Comment:
why have custom rules here. Shouldn't all modules use the same?

Reply:
Fixed in https://github.com/SAP/open-ux-tools/pull/257/commits/0ad205679050af6a49ee1e7619f136df131219a8